packages/melange-numeral/melange-numeral.0.0.1/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+synopsis: "Melange bindings for Numeral"
+description: "Melange bindings for the Numeral JavaScript library."
+maintainer: [
+  "Liubomyr Mykhalchenko <liubomyr.mykhalchenko@ahrefs.com>"
+  "Denis Strelkov <denis.strelkov@ahrefs.com>"
+]
+authors: [
+  "Liubomyr Mykhalchenko <liubomyr.mykhalchenko@ahrefs.com>"
+  "Denis Strelkov <denis.strelkov@ahrefs.com>"
+]
+license: "MIT"
+tags: ["melange" "org:ahrefs"]
+homepage: "https://github.com/ahrefs/melange-numeral/"
+bug-reports: "https://github.com/ahrefs/melange-numeral/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "3.8"}
+  "melange" {>= "2.0.0"}
+  "reason" {>= "3.10.0"}
+  "opam-check-npm-deps" {with-test} # todo: use with-dev-setup once opam 2.2 is out
+  "ocaml-lsp-server" {with-test}
```

Comment:
```suggestion
  "opam-check-npm-deps" {with-dev-setup}
  "ocaml-lsp-server" {with-dev-setup}
```

Reply:
I prefer `with-dev-setup` :) but then is there a way to install add these packages as deps while still using opam 2.1? I am imagining a way to set variables to true dynamically from the command line?

Reply:
ah nevermind i now understand what you were saying. Sorry for my misunderstanding. Mmh, i guess the `opam var` incantation above is fine as a temporary solution although if it's just for yourself i would encourage you to use opam 2.2 directly instead.